tasks/deploy-static.js

```diff
@@ -50,7 +50,9 @@ function task (opts) {
 
 	return Promise.all(files.map(function (file) {
 		return co(function*() {
-			const content = yield readFile(file, { encoding: 'utf-8' });
+			const imgFileExt = /\.(png|jpg|gif)$/i;
+			const isImage = imgFileExt.test(file);
```

Comment:
I've used this module in the past to ensure a file is infact an image, even if the extension was missing or incorrect -- https://www.npmjs.com/package/is-image